boards/arduino-atmega-common/include/arduino_pinmap.h

```diff
@@ -155,9 +158,27 @@ extern "C" {
 #define ARDUINO_PIN_A14         ARDUINO_PIN_68
 #define ARDUINO_PIN_A15         ARDUINO_PIN_69
 #endif
-
 /** @ */
 
+#define ARDUINO_A0              ADC_LINE(0)
+#define ARDUINO_A1              ADC_LINE(1)
+#define ARDUINO_A2              ADC_LINE(2)
+#define ARDUINO_A3              ADC_LINE(3)
+#define ARDUINO_A4              ADC_LINE(4)
+#define ARDUINO_A5              ADC_LINE(5)
+#define ARDUINO_A6              ADC_LINE(6)
+#define ARDUINO_A7              ADC_LINE(7)
+#ifdef CPU_ATMEGA2560
+#define ARDUINO_A8              ADC_LINE(8)
+#define ARDUINO_A9              ADC_LINE(9)
+#define ARDUINO_A10             ADC_LINE(10)
+#define ARDUINO_A11             ADC_LINE(11)
+#define ARDUINO_A12             ADC_LINE(12)
+#define ARDUINO_A13             ADC_LINE(13)
+#define ARDUINO_A14             ADC_LINE(14)
+#define ARDUINO_A15             ADC_LINE(15)
+#endif
+
```

Comment:
I still don't get this naming scheme, can you send a reference? Above, for instance, analog pins are named like `ARDUINO_PIN_AX`. In the Arduino forum it also appears that pins are addresses as `AX`.

Reply:
IIRC its partly _invented_ by @haukepetersen, see original PR by @dnahm #7227. This mapping is basically needed to support ADC in `sys/arduino/base.cpp`.

Reply:
What I didn't know/see was that `ARDUINO_PIN_AX` seems to be used for digital I/Os and `ARDUINO_AX` for analog inputs. Fine with me...

Reply:
right (I guess), otherwise there would be a name/macro clash between `readAnalog` and `readDigitial` and the corresponding pin mappings used in `sys/arduino/base.cpp`.